includes/class-wc-skroutz-analytics-tracking.php

```diff
@@ -248,4 +286,59 @@ private function calculate_tax_from_total($total, $tax_rate) {
 		return round( $total - $total / ( 1 + $tax_rate/100 ), 2 );
 	}
 
+	/**
+	 * Returns the paid by field to analytics
+	 *
+	 * @return string  The paid by field
+	 *
+	 * @since    1.7.0
+	 */
+	private function get_order_paid_by($payment_gateway) {
+		return apply_filters( 'wc_skroutz_analytics_tracking_order_paid_by_filter', $payment_gateway->id, $this->order, $payment_gateway );
+	}
+
+	/**
+	 * Returns the paid by description field to analytics
+	 *
+	 * @return string  The paid by description field
+	 *
+	 * @since    1.7.0
+	 */
+	private function get_order_paid_by_descr($payment_gateway) {
+		$paid_by_descr = mb_substr( $payment_gateway->get_title(), 0, self::PAID_BY_DESCR_MAX_LENGTH );
+		return apply_filters( 'wc_skroutz_analytics_tracking_order_paid_by_descr_filter', $paid_by_descr, $this->order, $payment_gateway );
+	}
+
+	/**
+	 * Returns the item name that should be reported to Analytics
+	 *
+	 * @return string  The item
+	 *
+	 * @since    1.7.0
+	 */
+	private function get_item_name($item, $product, $sa_product) {
+		return apply_filters( 'wc_skroutz_analytics_tracking_item_name_filter', $product->get_title(), $item, $product, $sa_product );
+	}
+
+	/**
+	 * Returns the item total price
+	 *
+	 * @return float item total price
+	 *
+	 * @since    1.7.0
+	 */
+	private function get_item_total($item, $product, $sa_product) {
```

Comment:
I think `$sa_product` contains the original wc_product as a variable. Do we want to pass `$product` as well for more clarity?

Reply:
Well the product variable is private, so we would need to change the visibility of the property. However on a second thought the sa_product does not provide valuable information for the item filters (name, price, quantity). So I suggest to drop it.